samples/tab-request-approval/nodejs/.env

```diff
@@ -0,0 +1,4 @@
+ClientId = 
```

Comment:
Do you need only these settings because I can see more settings in C# sample.

Reply:
Yes only this much settings are required. For c# sample some additional configurations are needed in appsettings file which is internally configured for nodejs sample